packages/dose3/dose3.5.0.1-2/opam

```diff
@@ -0,0 +1,56 @@
+opam-version: "2.0"
+maintainer: "pietro.abate@inria.fr"
+authors: [
+  "Pietro Abate"
+  "Jaap Boender"
+  "Roberto Di Cosmo"
+  "Johannes Schauer"
+  "Ralf Treinen"
+  "Stefano Zacchiroli"
+  "Jakub Zwolakowski"
+  "Olivier Rosello"
+]
+homepage: "http://www.mancoosi.org/software/"
+bug-reports: "https://gitlab.com/irill/dose3/-/issues"
+license: "LGPL-3.0-or-later WITH OCaml-LGPL-linking-exception"
+dev-repo: "git+https://gitlab.com/irill/dose3.git"
+build: [
+  ["./configure"]
+  [make "printconf"]
+  [make "libs" "apps"]
+]
+install: [make "installlib"]
+depends: [
+  "ocaml"
+  "ocamlgraph" {>= "1.8.6"}
+  "cudf" {>= "0.7"}
+  ("extlib" {>= "1.7.0" & < "1.7.8"} | "extlib-compat" {>= "1.7.0"})
+  "re" {>= "1.2.2"}
+  "ocamlfind" {build}
+  "ocamlbuild" {build}
+  "cppo" {build & >= "1.1.2"}
+]
+conflicts: "dose"
+patches: [
+  "0001-Install-mli-cmx-etc.patch"
+  "0002-dont-make-printconf.patch"
+  "0003-Fix-for-ocaml-4.06.patch" {ocaml:version >= "4.06.0"}
+  "0004-Add-unix-as-dependency-to-dose3.common-in-META.in.patch"
+  "0005-Fix-compatibility-with-ocamlgraph-2.0.patch"
+]
+synopsis: "Dose library (part of Mancoosi tools)"
+extra-files: [
+  [
+    "0004-Add-unix-as-dependency-to-dose3.common-in-META.in.patch"
+    "md5=618265012624df95902a98f756f1ca13"
+  ]
+  ["0003-Fix-for-ocaml-4.06.patch" "md5=877eedb18916f9e260525b1aee6da544"]
+  ["0002-dont-make-printconf.patch" "md5=a6e83acee4b55d35f5f30a8ef98df04f"]
+  ["0001-Install-mli-cmx-etc.patch" "md5=977b675e7e6e7ccc5d3d57534370c68c"]
+  ["0005-Fix-compatibility-with-ocamlgraph-2.0.patch" "md5=e17b0f0aaede654a19fb3f0e2e46c61a"]
+  ["0006-Grep-carefully-for-ocaml-system.patch" "md5=b9fd0f902ef8d65ab19792ef25d33974"]
```

Comment:
the checksum needs to be updated accordingly

Reply:
Sorry, I'm not paying enough attention. Done.